ql/src/test/results/clientnegative/msck_repair_1.q.out

```diff
@@ -21,4 +21,4 @@ POSTHOOK: Output: default@repairtable
 PREHOOK: query: MSCK TABLE default.repairtable
 PREHOOK: type: MSCK
 PREHOOK: Output: default@repairtable
-FAILED: Execution Error, return code 1 from org.apache.hadoop.hive.ql.ddl.DDLTask
+#### A masked pattern was here ####
```

Comment:
What's the real execption here?

Reply:
real exception is `FAILED: Execution Error, return code 40000 from org.apache.hadoop.hive.ql.ddl.DDLTask. org.apache.hadoop.hive.metastore.api.MetastoreException: MSCK is missing partition columns under file:/Users/tarak/IdeaProjects/tarak-Apache-Hive/itests/qtest/target/localfs/warehouse/repairtable/p1=c
`
since it contained file location details, it is getting masked in test case output as per [Masking rules](https://github.com/apache/hive/blob/5b76949da6fe65364a4e3766680871167131157f/itests/util/src/main/java/org/apache/hive/beeline/QFile.java#L336)